src/ui/screens/onboarding/OnboardingPrivacySlide.qml

```diff
@@ -41,15 +41,14 @@ ColumnLayout {
     }
 
     PrivacyFeaturesList {
-        Layout.topMargin: 24
+        Layout.topMargin: Qt.platform.os === "android" || Qt.platform.os === "ios" ? 32 : 24
```

Comment:
Is there a utility function available for `Qt.platform.isMobile`? If not, is it worth making one?